packages/zowe-explorer/package.json

```diff
@@ -1405,11 +1405,6 @@
           "command": "zowe.jobs.editSession",
           "group": "099_zowe_jobsProfileModification@0"
         },
-        {
-          "when": "view == zowe.jobs.explorer && viewItem =~ /^(?!.*_fav.*)server.*/",
-          "command": "zowe.jobs.removeJobsSession",
```

Comment:
Looks like this menu item was referencing a non-existent command, so makes sense to remove it 👍